crates/bench-api/src/lib.rs

```diff
@@ -211,9 +213,23 @@ impl BenchState {
 
         let mut linker = Linker::new(&self.store);
 
-        // Import a very restricted WASI environment.
+        // Create a WASI environment.
+
         let mut cx = WasiCtxBuilder::new();
         cx.inherit_stdio();
+        // Allow access to the current working directory so that the benchmark
+        // can read its input workload(s). The sightglass benchmark runner will
+        // make sure that this process is spawned with its current directory set
+        // to the current benchmark's directory.
+        let cwd_path = env::current_dir().context("failed to get the current working directory")?;
+        let cwd = File::open(&cwd_path).context("failed to open the current working directory")?;
```

Comment:
If you use `wasi_common::preopen_dir` it'll handle the Windows-specific logic to make this work.

Reply:
Thanks!